### DISCUSSION TITLE
Problem with irssi on M1

### DISCUSSION THREAD

**Discussion by jiphex** [Homebrew/discussions#1557] [Everyday usage] [answered]
I've installed `irssi` with `brew install irssi`.

When running it for the first time, I get:

```
❯ irssi
dyld: Library not loaded: /System/Library/Perl/5.28/darwin-thread-multi-2level/CORE/libperl.dylib
  Referenced from: /opt/homebrew/bin/irssi
  Reason: image not found
zsh: abort      irssi
```

Can someone confirm if they see the same issue, or if it's just me?

This is macOS 11.4, 13" MacBook Pro, M1

**Comment by nandahkrishna** [comment, marked as answer]
Could you check if `brew reinstall irssi` solves this now, after doing `brew update`? (This seemed similar to Homebrew/homebrew-core#77976, so I added a potential fix in https://github.com/Homebrew/homebrew-core/commit/0e830210aa2f7b97608ea2e4f41ad032e01a7af4).

**Reply by jiphex** [reply]
Yes, that seems to have resolved the issue - thanks so much!
